backend/src/core/common/port/cloudinary.ts

```diff
@@ -0,0 +1,6 @@
+import { UploadApiResponse } from 'cloudinary';
```

Comment:
Better to change to custom DTO with only needed fields. Port shouldn't know anything about the service which we use.